### PR TITLE
@task PDA-1411 & PDA-1432

### PR DESCRIPTION
**Pull request by lli-parasoft**

### CODE REVIEW

File: src/main/resources/i18n/aerospace/common.properties [hunk 1]
```diff
@@ -226,6 +226,16 @@ PASSWORD=Password:
 EXCHANGE=Exchange:
 REQUEST_QUEUE_ROUTING_KEY=Request queue routing key:
 RESPONSE_QUEUE_ROUTING_KEY=Response queue routing key:
+GRPC_CONFIGURATION_DETAILS_DESCRIPTION=The demo app comes pre-configured with host, port, and service names of gRPC and includes three methods:
+GRPC_CONFIGURATION_DETAILS=GRPC Configuration Details
```
Comment (yshi-parasoft) [inline src/main/resources/i18n/aerospace/common.properties:230]: Please use `gPRC` instead of `GRPC`

**Comment by yshi-parasoft** [comment]
Please follow the idea below to modify the content. We should also include description for Protobuf version.

The demo app comes with a JSON gRPC service which has the following configuration details:

Host: xxx
Port: xxx
Service Name: xxx

There is also a Protobuf version of the service with same methods:
Host: xxx
Port: xxx
Service Name: xxx

Here are the methods implemented in both service:

-------------------------------------------------------------------------------------------------------------------
|           Method           |         Description                                                                                                        |
--------------------------------------------------------------------------------------------------------------------
|  getStockByItemId     |  Unary call which sends item id and receives stock quantity                                    |
--------------------------------------------------------------------------------------------------------------------
|  getItemsInStock        |  Server streaming method which gets information of all items in stock in stream  |
--------------------------------------------------------------------------------------------------------------------
|  updateItemsInStock  |  Bidirectional streaming method which takes information to update item stock    |
|                                    |  and responds with updated item information                                                        |
--------------------------------------------------------------------------------------------------------------------

**Review by rwang-parasoft — CHANGES_REQUESTED** [review]
Need to change style of modal.

**Comment by yshi-parasoft** [comment]
image
Should apply style to align left for method names to be consistent.

**Comment by yshi-parasoft** [comment]
Also I think the service name for Protobuf is changed. Please update it.

**Review by rwang-parasoft — APPROVED** [review]

File: src/main/resources/static/common/js/demoAdmin.js [hunk 2]
```diff
@@ -109,6 +109,55 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
         }
     }
 
+    demo.loadGRPCConfigurationDetails = function() {
+        $http({
+            method: 'GET',
+            url: '/v1/demoAdmin/gRPCProperties',
+        }).then(function(result) {
+            const data = result.data.data
+            demo.gRPCConfigurationJsonDetails = [
```
Comment (yshi-parasoft) [inline src/main/resources/static/common/js/demoAdmin.js:118]: There are some UI lags when opening the modal dialog. I think the only information we need from the callback is the port. So please set other values before we make the API call and only set port value in the callback to improve the performance.

**Review by yshi-parasoft — CHANGES_REQUESTED** [review]
Please fix this before merging:
https://github.com/parasoft/parasoft-demo-app/pull/258/files#r1063712832

**Pull request merged**